### PR TITLE
feat: adds sqs configurator hook

### PR DESCRIPTION
**Pull request by srinidhis05**
What this PR does:
Adds the sqs configurator , which would take the queue details as input and create the queues in either localstack or AWS along with updating the corresponding secret based on config
TODO:

- [ ]  Add Readme for app usage 
- [ ]  Add helm hook job reference

### CODE REVIEW

File: hooks/sqs_configurator/config/app.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+queue:
+  q1:
+    name: card_create_srinidhi
```
Comment (venkatvghub) [inline hooks/sqs_configurator/config/app.yaml:3]: Change to something more generic and the associated key as well?

File: hooks/sqs_configurator/constants/constants.go [hunk 2]
```diff
@@ -0,0 +1,12 @@
+package constants
+
+const (
+	LOCALSTACK_ENDPOINT = "https://localstack-services.dev.razorpay.in"
+	LOCALSTACK_PREFIX   = "https://localstack-services.dev.razorpay.in/000000000000"
+	AWS_ENDPOINT        = "https://sqs.ap-south-1.amazonaws.com"
+	AWS_PREFIX          = "https://sqs.ap-south-1.amazonaws.com/141592612890"
```
Comment (venkatvghub) [inline hooks/sqs_configurator/constants/constants.go:7]: Remove account id please

File: hooks/sqs_configurator/package/secrets/kubernetes.go [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+	namespace string
+}
+
+//New initializes the client with the given namespace and secret
+func New(namespace string, secretName string, isLocal *bool) (KubeSecret, error) {
+	var config *rest.Config
+	if *isLocal {
+		home, exists := os.LookupEnv("HOME")
+		if !exists {
+			home = "/root"
```
Comment (venkatvghub) [inline hooks/sqs_configurator/package/secrets/kubernetes.go:32]: /root?

Comment (srinidhis05) [reply hooks/sqs_configurator/package/secrets/kubernetes.go:32]: This is for the local setup

Comment (srinidhis05) [reply hooks/sqs_configurator/package/secrets/kubernetes.go:32]: takes root as home if home env is not set

**Pull request merged**